examples/tic-tac-toe/README.md

```diff
@@ -0,0 +1,135 @@
+
+As in the shared protocol, once the game has ended, a `Trophy` is sent
+to any winning player. Unlike the shared protocol, only the admin can
+clean-up the Game once it has finished, because only they have access
+to it.
+
+``` mermaid
+sequenceDiagram
+    activate Player X
+    Player X->>Admin: new: Game
+    Player X->>Player X: [TurnCap]
+    deactivate Player X
+    Player X->>Game: send_mark: Mark
+    activate Admin
+    Admin->>Game: place_mark
+    Admin->>Player O: [TurnCap]
+    deactivate Admin
+    Player O->>Game: send_mark: Mark
+    activate Admin
+    Admin->>Game: place_mark
+    Admin->>Player X: [TurnCap]
+    deactivate Admin
+    Player X->>Game: ...
+    Admin->>Game: ...
+    Player O->>Game: ...
+    activate Admin
+    Admin->>Game: place_mark
+    Admin->>Player O: [Trophy]
+    deactivate Admin
+    Admin->>Game: burn
+```
+
+## Multi-sig tic-tac-toe
+
+The owned protocol avoids consensus, but it requires trusting a third
+party for liveness (The third party cannot make a false move, but it
+can choose not to place a move, or simply forget to). That third party
+may also need to run a service that keeps track of marks sent to games
+in order to apply them promptly, which adds complexity.
+
+There is an alternative approach, which leverages Sui's support for
+**multi-sigs** and **sponsored transactions**. Instead of entrusting
+the Game to a third party, it is sent to an address owned by a 1-of-2
+multisig, signed for by Player X and Player O.
+
+Play proceeds as in the owned protocol, except that the Admin is the
+multi-sig account. On each turn, the current player runs a transaction
+as themselves to send the mark, and then runs a transaction on behalf
+of the multi-sig to place it.
+
+Once play has finished, either player can run a transaction on behalf
+of the multi-sig account to `burn` the game. As the player is the
+sponsor, they will receive the storage rebate for performing the
+clean-up.
+
+The multi-sig account does not own anything other than the game object
+(it does not have any gas coins of its own), so the player sponsors
+the transaction, using one of its own gas coins.
+
+Sharing a resource while avoiding consensus by transfering it to a
+multi-sig account can be generalised from two accounts to up to ten
+(the limit being the number of keys that can be associated with one
+multi-sig).
+
+In order to create a multi-sig, the public keys of all the signers
+needs to be known. Each account address on Sui is the hash of a public
+key, but this operation cannot be reversed, so in order to start a
+multi-sig game, players must exchange public keys instead of
+addresses.
+
+## Pros and cons
+
+The shared protocol's main benefit is that its on-chain logic and
+client integration are straightforward, and its main downside is that
+it relies on consensus for ordering.
+
+In contrast, the owned protocol uses only fast-path transactions (no
+consensus), but its on-chain logic is more complicated because it
```

Comment:
Maybe use this parenthetical in the first note (no consensus) and then not needed here.